### ISSUE TITLE
Chrono tanks do not chrono immediately on force move on playtest

### ISSUE THREAD

**Issue by Punsho** [OpenRA/OpenRA#17058]
Instead they move to the place they were ordered and chrono only when in range. IIRC this behaviour should have been only present for queued chrono orders and nor for normal ones

**Comment by matjaeck** [comment]
This is the expected behavior per #16129. 

> If ordered to teleport outside of its range, the unit will now first move into teleport range and then teleport.

**Comment by Punsho** [comment]
So I recalled correctly, It should have only affected queued orders. The current behaviour makes it much harder to retreat and is not wanted.

**Comment by abcdefg30** [comment]
No, it was meant to affect all orders, this is intentional, but not unchangeable (https://github.com/OpenRA/OpenRA/pull/16129#pullrequestreview-213361118).

**Comment by netnazgul** [comment]
Yes, I remember we had a convo about it, and it's either ctanks always do a jump and then move to a location or the other way around - move and then jump. The first one is better for retreating, the last one is better for them to get into the desired position. Though the problem with the last one is that a group of ctanks won't jump simultaneously if ordered from a long enough position - they will drive closer one by one and arrive in packs. Which also might be less preferable.

**Comment by Punsho** [comment]
In case my point wasn't clear what I'm saying here the whole time boils down to. Why not both? Chrono first on normal force move. Move into position first on queued force move

**Comment by PunkPun** [comment]
Ping, it's still a hot issue. The behaviour I was describing is that same one implemented in SC2 for stalkers

**Comment by deleted-user-1** [comment]
Chronotanks have been changed since then so the behavior to opt for quick reaction instead of precision is likely the better choice now.

**Comment by netnazgul** [comment]
What I've just thought is that in case of "jump-then-move" it would be best to have a jump radius visible; this can be tied to holding "Alt".

**Comment by abcdefg30** [comment]
https://github.com/OpenRA/OpenRA/pull/18581#issuecomment-799002352 details the latest idea on how we want to implement this.
